frontend/cypress/pages/metrics.ts

```diff
@@ -54,6 +61,17 @@ class Metrics {
   goMetricsStep() {
     cy.contains('Next').click()
   }
+
+  selectCrewSetting() {
+    cy.wait('@verifyJira').then((currentSubject) => {
+      cy.contains(`${currentSubject.response.body.users.join(', ')}`).should('exist')
+    })
+
+    cy.contains('Included Crews').siblings().click()
+
+    cy.get("[type='checkbox']").should('be.checked')
+    cy.get('div.MuiBackdrop-root.MuiBackdrop-invisible.MuiModal-backdrop').click({ force: true })
```

Comment:
Can we use another method to get the element, 
classname is not stable for e2e testing when upgrading the 3rd package